### ISSUE TITLE
lsp-ui-peek-find-references throws errors

### ISSUE THREAD

**Issue by lassemaatta** [emacs-lsp/lsp-ui#706]
I updated `lsp-ui` this morning, and now when running `lsp-ui-peek-find-references` on a symbol in some clojure code, I get

```
lsp-ui-peek--render: Invalid function: lsp-ui--mute-apply
```
and sometimes

```
lsp-ui--workspace-path: Wrong type argument: arrayp, nil
lsp-ui-peek--render: Invalid function: lsp-ui--mute-apply
```

Using: `lsp-ui 20220509.1305` & `lsp-mode 20220505.630`

I would guess this is somehow related to [this](https://github.com/emacs-lsp/lsp-ui/commit/ae1801e76a6682b6e52f9061834f4efdaec47b00) commit.

**Comment by jcs090218** [comment]
Hi, can you try reinstall `lsp-ui`? The macro is defined in `lsp-ui.el` file.

**Comment by lassemaatta** [comment]
I tried to `(unload-feature lsp-ui)` + `(package-reinstall lsp-ui)` + restarted emacs: still got the same error.

**Comment by jcs090218** [comment]
Okay, I think `lsp-ui` is requiring module the other way around. Should be fixed in #707.

**Comment by jcs090218** [comment]
I have first revert the changes! Sorry for the inconvenience!
